arbor/morph/embed_pwlin.cpp

```diff
@@ -275,11 +275,11 @@ embed_pwlin::embed_pwlin(const arb::morphology& m) {
             segment_cables_[seg.id] = mcable{bid, pos0, pos1};
         }
 
-        double length_0 = parent==mnpos? 0: data_->length[parent].back().second[1];
+        double length_0 = parent==mnpos? 0: data_->length[parent].back().element[1];
```

Comment:
... and here the subscript operator. Any particular reason?

Reply:
And here, it is accessing the interpolant value directly (for length, there are two elements, and `[1]` refers to the value at the end). The difference here is that we want the exact value used for interpolation without relying upon any further calculations, to avoid any potential rounding issues.

Reply:
👍